### PR TITLE
 Lesson-3-1 CSS draft

### PR DESCRIPTION
**Pull request by leipei92**
Styling Todo list application wth CSS modules.

### CODE REVIEW

File: src/TodoListItem.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from "react";
+import style from "./TodoListItem.module.css";
 
 function TodoListItem({ todo, onRemoveTodo }) {
     return (
-        <li>{todo.title} <button onClick={() => onRemoveTodo(todo.id)}>Remove</button></li>
+        <li className={style.ListItem}>{todo.title} <button onClick={() => onRemoveTodo(todo.id)} className={style.button}>Remove</button></li>
```
Comment (JoHeyJo) [inline src/TodoListItem.js:6]: Good job naming your styles after your components, this will be important as your app grows.

**Review by JoHeyJo — APPROVED** [review]
Nice job, app is looking great! Something to be cautious about. There is a lot of mixing and matching of responsive and non-responsive units throughout your styles. Be very deliberate about using non-responsive styles.

**Comment by leipei92** [comment]
> Nice job, app is looking great! Something to be cautious about. There is a lot of mixing and matching of responsive and non-responsive units throughout your styles. Be very deliberate about using non-responsive styles.

That's true! I was just throwing different options to make it work. I will work on that!

**Comment by JoHeyJo** [comment]
That excellent! It's good to play around and get hands-on experience.

**Pull request merged**